controllers/example.js

```diff
@@ -21,19 +23,13 @@ exports.postExample = async (req, res) => {
 
 exports.deleteExample = async (req, res) => {
   const exampleId = req.params.exampleId
-  try {
-    const example = await Example.findById(exampleId)
+  const example = await Example.findById(exampleId)
```

Comment:
remove that line and move if statement under the line 32, no point it doing here `findById` if `findByIdAndRemove` does the same under the hood

Reply:
done)